firmware/Core/Src/main.c

```diff
@@ -106,6 +106,37 @@ const osThreadAttr_t TASK_execute_telecommands_Attributes = {
   .priority = (osPriority_t) osPriorityNormal,
 };
 
+osThreadId_t TASK_monitor_freertos_highstack_watermarks_Handle;
+const osThreadAttr_t TASK_monitor_freertos_highstack_watermarks_Attributes = {
+  .name = "TASK_monitor_freertos_highstack_watermarks",
+  .stack_size = 1024,
+  .priority = (osPriority_t) osPriorityBelowNormal6,
+};
+
+const Task_Info_t task_handles_array [] = {
+  {
+    .task_handle = &defaultTaskHandle,
+    .stack_size_bytes = 128 * 4
```

Comment:
Instead of duplicating the stack size values here, what do you think about storing a pointer to the attributes struct and reading it from there? I think that might be a bit cleaner?

Reply:
Sure thing, let me attempt this approach